docs/en/Mounting-Alluxio-FS-with-FUSE.md

```diff
@@ -115,6 +111,12 @@ These are the configuration parameters for Alluxio-FUSE.
 {% endfor %}
 </table>
 
+## Building manually
+
+You can build Alluxio-FUSE with the maven profile `fuse`. This profile will be automatically activated by maven when it is detected that you are building Alluxio with a JDK version 8 or newer.
```

Comment:
this `fuse` module is now built by default with Alluxio distribution. Guess unnecessary for this section now?